app/src/main/java/com/kuit/kuit6android/ui/home/data/Restaurant.kt

```
@@ -0,0 +1,3 @@
+package com.kuit.kuit6android.ui.home.data
+
+data class Restaurant()
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)